src/components/DownloadAirgapBundle.js

```diff
@@ -82,13 +85,7 @@ cat install.sh | sudo bash -s airgap
                   <div className="FormLabel u-marginBottom--5"> Download airgap installer </div>
                   <span className="u-fontSize--small u-fontWeight--normal u-color--scorpion u-lineHeight--normal u-marginBottom--more"> You can get the airgap bundle a couple of ways. You can download it directly from here or via a CLI. </span>
                   <div className="u-marginTop--normal">
-                    <button
-                      type="button"
-                      className="Button primary"
-                      onClick={() => this.handleDownloadBundle()}
-                    >
-                      Download airgap bundle
-                    </button>
+                  <a href={downloadUrl}> Download airgap Bundle </a>
```

Comment:
style is changed here though.

Reply:
What is the name of the file that is downloaded?

Reply:
It is either `later.tar.gz` or `<sha>.tar.gz`

![Screen Shot 2022-06-28 at 8 43 55 AM](https://user-images.githubusercontent.com/27239392/176223654-a363d911-a84c-4738-8f7f-31e5173dfdf3.png)

Reply:
Great